### PR TITLE
 adding docker sandbox flags to 0.14.1 bazelrc

### PR DESCRIPTION
**Pull request by nlopezgi**

### CODE REVIEW

File: bazelrc/bazel-0.14.1.bazelrc [hunk 1]
```diff
@@ -92,3 +94,14 @@ build:results-local --experimental_remote_spawn_cache
 build:results-local --remote_cache=remotebuildexecution.googleapis.com
 build:results-local --remote_timeout=3600
 build:results-local --bes_results_url="https://source.cloud.google.com/results/invocations/"
+
+# The following flags are only necessary for local docker sandboxing
+# with the rbe-ubuntu16-04 container. Use of these flags is still experimental.
+build:docker-sandbox --host_javabase=@bazel_toolchains//configs/ubuntu16_04_clang/1.0:jdk8
+build:docker-sandbox --javabase=@bazel_toolchains//configs/ubuntu16_04_clang/1.0:jdk8
+build:docker-sandbox --crosstool_top=@bazel_toolchains//configs/ubuntu16_04_clang/1.0/bazel_0.13.0/default:toolchain
```
Comment (xingao267) [inline bazelrc/bazel-0.14.1.bazelrc]: why 0.13.0?

File: bazelrc/bazel-0.14.1.bazelrc [hunk 2]
```diff
@@ -92,3 +94,14 @@ build:results-local --experimental_remote_spawn_cache
...
+build:docker-sandbox --experimental_docker_image=gcr.io/cloud-marketplace/google/rbe-ubuntu16-04@sha256:59bf0e191a6b5cc1ab62c2224c810681d1326bad5a27b1d36c9f40113e79da7f
+build:docker-sandbox --spawn_strategy=docker --strategy=Javac=docker --genrule_strategy=docker
```
Comment (xingao267) [inline bazelrc/bazel-0.14.1.bazelrc]: Put into separate lines?

File: bazelrc/bazel-0.14.1.bazelrc (hunk 1 above)
Comment (nlopezgi) [reply bazelrc/bazel-0.14.1.bazelrc]: copy paste error, fixed

File: bazelrc/bazel-0.14.1.bazelrc (hunk 2 above)
Comment (nlopezgi) [reply bazelrc/bazel-0.14.1.bazelrc]: done

**Pull request merged**